feature/home/src/main/java/io/filmtime/feature/home/HomeScreen.kt

```diff
@@ -101,6 +153,80 @@ fun VideoSectionRow(
   }
 }
 
+@Composable
+fun UnknownErrorScreen() {
+  Column(modifier = Modifier.fillMaxSize(),
+    horizontalAlignment = Alignment.CenterHorizontally,
+    verticalArrangement = Arrangement.Center){
```

Comment:
Is it necessary for this column to contain only this Text? Can we move modifiers to Text?